cmake/macros/testWrapper.py

```diff
@@ -379,9 +408,14 @@ def _runCommand(raw_command, stdout_redir, stderr_redir, env,
 
     # If desired, diff the provided file(s) (must be generated by the test somehow)
     # with a file of the same name in the baseline directory
+    failuresDir = None
+    if args.failures_dir:
+        failuresDir = args.failures_dir.replace('<PXR_CTEST_RUN_ID>',
```

Comment:
If PXR_CTEST_RUN_ID isn't set in the environment this will generate an invalid path with "<", ">" in it. It was recommended to just drop "<", ">"?

Reply:
Done:
https://github.com/pmolodo/USD/commit/c24838f63735082c515e7dc8b99032397c949c72#diff-20f6c1250283d4110c2cacd19f922472ce55350c2e19d29f9491db6b2e792793R422